internal/mqini/mqini.go

```diff
@@ -76,3 +88,168 @@ func GetErrorLogDirectory(qm *QueueManager) string {
 	}
 	return filepath.Join(qm.Prefix, "qmgrs", qm.Directory, "errors")
 }
+
+//Update qm.ini file with the user supplied stanzas.
+func AddStanzas(qmname string) error {
+
+	qm, err := GetQueueManager(qmname)
+	if err != nil {
+		return err
+	}
+
+    //Find the ini file
+    files := getIniFileList()
+	
+	//If we are given ini file, read it.
+    for _, infile := range files {
+        iniFileBytes, err := ioutil.ReadFile(infile)
+		if err != nil {
+			return err
+		}
+		userconfig := string(iniFileBytes)
+
+	    //No ini file supplied, so nothing to do.
+	    if len(userconfig) == 0 {
+		    continue
+	    } else {
+			//find the corresponding qmgrs config file.
+			inifilepath := GetIniFilePath(infile, qm)
+			if err != nil {
+				return err
+			}
+			iniFileBytes, err := ioutil.ReadFile(inifilepath)
+			if err != nil {
+				return err
+			}
+			//read the initial version.
+			qmconfigStr = string(iniFileBytes)
+			if err != nil {
+				return err
+			}
+			//Update the qmgr ini file with user config.
+            WriteToIniFile(userconfig, inifilepath)
+        }
+	}
+	return nil
+}
+
+// read through /etc/ and check if user provided any .ini file 
+// to update.
+func getIniFileList() []string {
+
+    fileList := []string{}
+    filepath.Walk("/etc", func(path string, f os.FileInfo, err error) error {
+        if strings.HasSuffix(path, ".ini") {
+            fileList = append(fileList, path)
+		}
+		return nil       
+	})
+	return fileList
+}
+
+// Based on the ini file(qm.ini or mqs.ini or mqat.ini), return corresponding
```

Comment:
Godoc comments usually follow a pattern which includes the name of the function or package your describing.  See [this blog entry on documenting Go code](https://blog.golang.org/godoc-documenting-go-code)